index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Recipe Website</title>
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>Personal Website Recipe</h1>
+    </header>
+    <main>
+        <div class="main">
+            <img style="max-height: 408px;" src="/files/chicken-pot-pie-crust.jpg" alt="chicken-pot-pie-crust">
+            <div class="main__article">
+                <h2>Chicken Pot Pie</h2>
+                <span>When you think of comfort food, what comes to mind?
+                    Chicken Pot Pie of course! It's warm, rich and creamy, loaded with vegetables and chicken and topped with a tasty golden crust. It's a perfect, healthy dinner recipe for events or large family gatherings. And this dish makes for excellent leftovers too, if you're lucky enough to have any!
+                    This recipe has a long list of ingredients, but it is a very easy meal to put together. It is full of healthy, nutritive foods from our approved foods list, and it's a really tasty, balanced meal. Feel free to make this dish your own by substituting other vegetables and herbs. Enjoy!
+                </span>
+                <div class="container__p">
+                    <p class="">PREP TIME</p>
+                    <p class="">COOK TIME</p>
+                    <p class="">TOTAL TIME</p>
+                    <p class="">20 min</p>
+                    <p class="">20 min</p>
+                    <p class="">40 min</p>
```

Comment:
empty class attribute. Better to remove these

Reply:
thanks, forgot to remove